workflow-bot-app/src/test/java/com/symphony/bdk/workflow/SendMessageIntegrationTest.java

```diff
@@ -54,13 +57,34 @@ void sendMessageOnMessage() throws Exception {
     engine.deploy(workflow);
     engine.onEvent(messageReceived("/message"));
 
-    verify(messageService, timeout(5000)).send(anyString(), any(Message.class));
+    ArgumentCaptor<Message> captor = ArgumentCaptor.forClass(Message.class);
+    verify(messageService, timeout(5000)).send(anyString(), captor.capture());
+    Message sent = captor.getValue();
+    assertThat(sent.getData()).contains("id", "123456");
 
     assertThat(workflow).isExecuted()
         .hasOutput(String.format(OUTPUTS_MSG_KEY, "sendMessage1"), message)
         .hasOutput(String.format(OUTPUTS_MSG_ID_KEY, "sendMessage1"), message.getMessageId());
   }
 
+  @Test
+  void sendMessageOnMessage_withException_errorVarNotEmpty() throws Exception {
```

Comment:
For me this test in not related to the SendMessage activity, so I would put it somewhere else. Maybe here https://github.com/finos/symphony-wdk/blob/master/workflow-bot-app/src/test/java/com/symphony/bdk/workflow/VariablesIntegrationTest.java. or https://github.com/finos/symphony-wdk/blob/master/workflow-bot-app/src/test/java/com/symphony/bdk/workflow/ErrorIntegrationTest.java..